### ISSUE TITLE
Docs on running second site on same domain

### ISSUE THREAD

**Issue by digininja** [Raikia/FiercePhish#70]
I might have set things up wrong to get in this position but I set the app up on targetdomain.com and now need to host some files on the domain so that I can direct the users to them.

I've moved the app out of the way and have now had to edit the laravel routes file to allow me to host some files in the root of the domain.

As an example I've added

```
Route::get('/amazon', function () {
    return 'Amazon landing page';
});


Route::get('/', function () {
    return Redirect::to('https://targetdomain.com/', 301); 
});
```

It would be nice to have a wiki page on how to do this correctly as this feels like a hack that will go wrong next time I update the app and you overwrite the routes/web.php file.

**Comment by Raikia** [comment]
Yeah, I do agree with this.  This is one of the reasons I actually like running FP on a different port as well, because it makes it a bit easier to handle (you can configure the port to run it on in the installer).  That said, the installer is created to allow you to rerun it to reconfigure everything without breakage so that might be a solution for you.

But yeah, Laravel clobbers entire hosting directories, so the solution is really within the installer and configuration of Apache.  I'll include this in v2.0, good feedback

**Comment by digininja** [comment]
So if re-run the installer and pick a different port, will it clear
anything out that I've set up so far?

I've nothing really important in, just test stuff, so doesn't matter if I
do lose things.

I know Apache config files fairly well, I could just edit the config file
if that would be easier but don't know if anything else needs updating in
the app config.

**Comment by Raikia** [comment]
You should be able to rerun the installer with no issues and not lose any data....

Or you can simply edit "/etc/apache2/sites-available/fiercephish.conf", change the :80 to :<port>

And then edit "/etc/apache2/ports.conf" and insert a "Listen <port>" line.

Then restart Apache.

**Comment by digininja** [comment]
That worked nicely, I've moved FiercePhish over to a different port and now
have my app running on 80.

When I run my next phish for a different company, will I need to re-run the
installer to change all the domain details?

**Comment by Raikia** [comment]
Yes, thats how I made it work.  v2.0 will allow you to configure for multiple domains because thats a thing I didn't expect people to need to do frequently but turned out it is needed.

**Comment by digininja** [comment]
Makes sense for out sourced testers who have different clients each week.
